CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic
+Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
+
+## [Unreleased]
+
+### Added
+
+- Add Coveralls support
+- Add testing with Tarantool 2.x-latest
+- Add reusable testing workflow (integration testing with latest Tarantool)
+- Add simple CI based on GitHub actions (#114)
```

Comment:
Issue refs are not clickable and it's rather disappointing.
(I've checked tarantool-python CHANGELOG and they're not clickable either. So it seems that it's normal.)